packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/Messages.md

```diff
@@ -79,7 +79,7 @@ You can add actions to a message, to allow users to interact with the message co
 
```

Comment:
Could we take this pr as an opportunity to change: _### Message**s** actions_ to _### Message actions_

It's been bothering me that I didn't go with that format to begin with loo

Reply:
No problem!